src/App.js

```diff
@@ -1,5 +1,41 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../src/constants/messages.js";
+import carInputValidation from "./utils/carInputValidation.js";
+import countInputValidation from "./utils/countInputValidation.js";
+import progress from "./utils/progress.js";
+import racePrint from "./utils/racePrint.js";
+import winner from "./utils/winner.js";
 class App {
-  async play() {}
+  async play() {
+    MissionUtils.Console.print(MESSAGES.START);
+    const CARS_NAMES = await MissionUtils.Console.readLineAsync("");
+    carInputValidation(CARS_NAMES);
+
+    MissionUtils.Console.print(MESSAGES.ANSWER_COUNT);
+    const COUNT = await MissionUtils.Console.readLineAsync("");
+    countInputValidation(COUNT);
+
+    //게임 진행
+    MissionUtils.Console.print("\n" + MESSAGES.RESULT);
+    const CARS = CARS_NAMES.split(",");
```

Comment:
```suggestion
    const CARS = CARS_NAMES.split(","); // carInputValidation 함수보다 위로 올리기
```
배열로 바꾸는 로직이 `carInputValidation` 함수에도 포함되어 있어서 이 로직을 위로 올리고 `carInputValidation`에 인자로 `CARS`를 넘겨주면 `carInputValidation` 함수 안에서는 배열로 바꾸지 않아도 될 것 같아요!